data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,470 @@ mission "Unfettered returning home"
+			`	"I even see technology from those savage Korath that we fought valiantly to obtain!"`
+				to display
+					has "outfit: Korath Detainer"
+			`	Her tone suddenly changes as she raises her voice for everyone to hear. "I, Mera, challenge you! The winner will earn the right to own this honorable relic and its mighty weapons!"`
+			choice
+				`	"What do I get if I win?"`
+					goto win
+				`	"Can't I just sell it to you?"`
+				`	"I bought this ship. It's mine and I shall do what I please with it. I'm leaving now."`
+					goto headingout
+			`	"No, because in the eyes of my brethren that would mean I, too, did not deserve this ship. They would then challenge me for it."`
+			label win
+			`	"You will win the right to access our worlds. In order to keep our respect, this Anomalocaris may not be destroyed or sold, until you prove yourself another way. This is a priceless artifact. Should you refuse to prove your worth, you will no longer be allowed to land here without paying a fee.`
+			`	"Beware, for there are no official rules, other than the fact that you may only use this one ship. So, will you take to the skies, or are you too afraid?"`
+			choice
+				`	"I accept."`
+				`	"I have nothing to prove to you. If you want it, you'll have to chase me down. No duel."`
+					goto headingout
+			`	"Good! Of course, I won't be fighting you personally, as it is too risky to do so. I am but an engineer, after all. I would also not want to deny my brethren the opportunity to prove themselves against a mighty human warrior!"`
+			`	You're not sure if Mera is being serious or not, but her friends seem to be in for it. Their leader approaches you. "So you think you deserve to fly this mighty ship of the Hai empire?"`
+			`	"Yeah, what makes you think that, human?" another one adds.`
+			choice
+				`	(Say nothing.)`
+				`	"Are you trying to scare me out of the fight?"`
+					goto scare
+			`	"I guess we'll find out," says a third.`
+			label scare
+			`	The leader of the group gets closer and addresses you in a more serious tone. "You should be aware that this is not so much a friendly competition as it is a fight for you to truly earn the right to fly this ship. You should know that this fight has a few... implicit rules. We would assume everyone knows them, but since you are an outsider I will explain them to you. Naturally, since this is a duel nobody other than us may interfere, but also know that being disabled counts as a loss. Don't even think about tricking us by switching to something other than this Anomalocaris, or we will not bother showing up! You must prove you deserve it, after all.`
```

Comment:
```suggestion
			`	The leader of the group gets closer and addresses you in a more serious tone. "You should be aware that this is not so much a friendly competition as it is a fight for you to truly earn the right to fly this ship. You should know that this fight has a few... implicit rules, even if there are no official ones. We would assume everyone knows them, but since you are an outsider I will explain them to you." He then walks beside you.`
			`Naturally, this is a duel nobody other than us may interfere, but also know that being disabled counts as a loss. Don't even think about tricking us by switching to something other than this Anomalocaris, or we will not bother showing up! You must prove you deserve it, after all.`
```
Splitting up this a bit since it does drag on a little as a quote. Also, you just got done saying there are no rules. But now there are implicit ones? Added a bit referencing that.